vms/platformvm/txs/mempool/mempool.go

```diff
@@ -224,26 +233,33 @@ func (m *mempool) HasTxs() bool {
 }
 
 func (m *mempool) PeekTxs(maxTxsBytes int) []*txs.Tx {
-	txs := m.unissuedDecisionTxs.List()
-	txs = append(txs, m.unissuedStakerTxs.List()...)
-
-	size := 0
-	for i, tx := range txs {
-		size += len(tx.Bytes())
-		if size > maxTxsBytes {
-			return txs[:i]
-		}
+	txs := make([]*txs.Tx, 0)
```

Comment:
Why are we allocating an empty array here (rather than just using a `nil` slice?)